src/layers/Dialog/Dialog.module.css

```diff
@@ -41,10 +41,18 @@
     padding: 20px 20px 10px 20px;
 
     .headerText {
-      margin: 0;
-      padding: 0;
       flex: 1;
       display: inline-flex;
+
+      h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6 {
```

Comment:
wasn't sure about this, but it feels like it should be ok to do

Reply:
This would override any body styles which they might want making it somewhat of a breaking change. I would recommend we not do this.